### ISSUE TITLE
[1.12] ClassNotFoundException on BlockOres2's getDrops

### ISSUE THREAD

**Issue by bafomdad** [defeatedcrow/HeatAndClimateMod#27]
Stacktrace:
```java.lang.NoSuchMethodError: net.minecraft.world.IBlockAccess.func_180494_b(Lnet/minecraft/util/math/BlockPos;)Lnet/minecraft/world/biome/Biome;
    at defeatedcrow.hac.main.block.ores.BlockOres2.getDrops(BlockOres2.java:161)
```

The line it crashes on: https://github.com/defeatedcrow/HeatAndClimateMod/blob/1.12.1_master/java/defeatedcrow/hac/main/block/ores/BlockOres2.java#L161

This crashes because it is called from the server, and this getBiome method in IBlockAccess is a clientside-only method. A better method would be to use `world.getBiomeForCoordsBody`, which is not clientside-only

**Comment by defeatedcrow** [comment]
Thanks for the report!
It fixed in 7ad3c14.

**Comment by bafomdad** [comment]
works now, thanks for the quick fix + release!
